compiler/compile/OMRCompilation.hpp

```diff
@@ -704,6 +704,12 @@ class OMR_EXTENSIBLE Compilation
    void verifyBlocks(TR::ResolvedMethodSymbol *s = 0);
    void verifyCFG(TR::ResolvedMethodSymbol *s = 0);
 
+   typedef TR::typed_allocator<std::pair<const TR::Node *, TR::TreeTop *>, TR::Region &> NodeTreeTopPairAllocator;
```

Comment:
Type given to `typed_allocator` must match type given to `std::list`.

```suggestion
   typedef TR::typed_allocator<std::pair<TR::Node *, TR::TreeTop *>, TR::Region &> NodeTreeTopPairAllocator;
```

Reply:
corrected.